enf-ssh/Makefile

```diff
@@ -0,0 +1,51 @@
+VERSION := 1.0.0
+ORG := xaptum
+NAME := enf-ssh
+IMG := $(ORG)/$(NAME)
+
+## Override the on the command line
+## make USER=... ADDRESS=... DNS=...
```

Comment:
Should the DNS setting be mentioned in the README?

Reply:
Yes, thanks.  I didn't want to complicate this with instructions for setting up DNS, but should mention that variable.